modules/find-policy-violations.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-cluster-configuration.adoc
+:_module-type: PROCEDURE
+[id="find-policy-violations_{context}"]
+= Finding policy violations
+
+[role="_abstract"]
+The *Policy Violations by Severity* widget in the *Configuration Management* view displays policy violations in a sunburst chart.
+Each level of the chart is represented by one ring or circle.
+
+* The innermost circle represents the total number of violations.
+* The next ring represents policy categories (*Low*, *Medium*, *High*, and *Critical*).
```

Comment:
You could remove the parentheses from this line as follows:

"* The next ring represents the **Low**, **Medium**, **High**, and **Critical** policy categories."